README.md

```diff
@@ -68,6 +74,7 @@ Steps (assumes a linux system)
 
   `http://localhost:8000/approval_polls`
 
+  (However, if you want Django to run on a public IP, make sure you update the ALLOWED_HOSTS variable to include the required IP address in your local_settings.py.)
```

Comment:
Sure, I have added the link to the documentation and the note as required. 

Thanks!